scripts/develop.sh

```diff
@@ -157,6 +157,7 @@ echo " Require ${REPOSITORY_NAME} using lando composer "
 echo "--------------------------------------------------"
 
 $LANDO composer config repositories.${INSTALL_PROFILE_DIRECTORY} '{"type": "path", "url": "/'${INSTALL_PROFILE_DIRECTORY}'", "options": {"symlink": true}}'
+$LANDO composer config repositories.ecms_patternlab '{"type": "package", "package": {"name": "state-of-rhode-island-ecms/ecms_patternlab", "version": "1.0.0", "type": "pattern-lab", "source": {"url": "https://github.com/State-of-Rhode-Island-eCMS/ecms_patternlab", "type": "git", "reference": "master"}}}'
```

Comment:
I think we need to update the README with this information too. The distribution will need to make this composer addition as well. Even though it is listed as a requirement for the profile, I _believe_ the distribution won't know where to place the file.

Reply:
I am working on a feature branch within the ecms_distribution. Pending results of that work I will update documentation and configuration as needed.